src/stores/UserStore.ts

```diff
@@ -0,0 +1,31 @@
+
+import type { GetUsersDto } from "@/dtos/GetUsersDto";
+import { UserService } from "@/services/UserService";
+import { defineStore } from "pinia";
+
+const userService: UserService = new UserService();
+
+export const UserStore = defineStore({
+  id: "userStore",
+  state: () => ({
+    loggedInUser: { username: "" } as GetUsersDto,
+  }),
+  getters: {
+    userName: (state) => {
+      if (state.loggedInUser.username != undefined) return state.loggedInUser.username;
+      else return "";
+    },
+  },
+  actions: {
+  /*  createUserandSignUp(username: string, password: string) {
+      userService
+        .createUserandSignUp(username, password)
+        .then((user) => (this.loggedInUser = user))
+        .catch((err) => console.log(err));
+    }, */
```

Comment:
if we don't need that maybe let's delete?

Reply:
Sure!